### ISSUE TITLE
Support Travis-CI build stages

### ISSUE THREAD

**Issue by Framstag** [Framstag/libosmscout#333]
See https://blog.travis-ci.com/2017-05-11-introducing-build-stages

**Comment by aquiles2k** [comment]
Isnt this only for enterprise travis?

**Comment by Framstag** [comment]
I have to admit, that I was not sure, while reading the blog article.

**Comment by Framstag** [comment]
Travis supports steps which are executed in order. Each step can have multiple jobs that can be executed in parallel. There is no data stored in between step execution. So building and executing tests based on the build result cannot be two steps.

Since I see not practical need for this feature now => won't fix.
